### DISCUSSION TITLE
Guild#Member going to be depreciated in v13?

### DISCUSSION THREAD

**Discussion by BoredFishRE** [discordjs/discord.js#5550] [Q&A] [answered]
So I just heard that Guild#Member was going to be depreciated when v13 comes out. Does this mean something like `message.guild.members.cache.get('695345708638797865');` will be depreciated? I want to know because the bot I'm developing needs to last for awhile, and it can't really have too much downtime.

**Comment by DaStormer** [comment, marked as answer]
Guild#member will be removed, so you wouldn't be able to do something like `guild.member()`. However, `guild.members.cache.get()` will still work.

**Reply by BoredFishRE** [reply]
Ah thanks! Was worried that I would basically have to redo a lot of stuff. Thanks!
